draft-ietf-rats-corim.md

```diff
@@ -923,8 +927,8 @@ The following describes each member of the `measurement-values-map`.
   {{sec-comid-svn}}.
 
 * `digests` (index 2): Contains the digest(s) of the measured environment
-  together with the respective hash algorithm used in the process.  See
-  {{sec-common-hash-entry}}.
+  together with the respective hash algorithm used in the process.
```

Comment:
Squash single sentence into 1 line.

Reply:
This will be done in future PR.

Reply:
I'll let Henk who volunteered to make a pass over the whole document to "onelinearise" it